test/publish/test_orcid.py

```diff
@@ -109,8 +130,8 @@ def test_export_author_orcids(
     assert result is True
 
     with test_reports_session.begin() as session:
-        rows = session.query(AuthorOrcids).all()
-        assert len(rows) == 1
+        rows = session.query(AuthorOrcids).order_by(AuthorOrcids.sunetid).all()
```

Comment:
added the sort since query results aren't guaranteed to be deterministically ordered otherwise, and flappy tests are no fun